mods/cnc/maps/n64gdi01/n64gdi01.lua

```diff
@@ -0,0 +1,201 @@
+	destroyBiotechCenterObjective = player.AddPrimaryObjective("Destroy the biotech facility.")
+
+	Trigger.OnAllKilled(SamSites, function()
+		AirSupport = Actor.Create("airstrike.proxy", true, { Owner = player })
+		AirSupportEnabled = true
+		player.MarkCompletedObjective(destroySAMsCenterObjective)
+	end)
+
+	Trigger.OnDamaged(Obelisk01, function()
+		Trigger.AfterDelay(DateTime.Seconds(1), Obelisk01.Kill)
+	end)
+
+	Trigger.OnAllKilled(Obelisk, function()
+		player.MarkCompletedObjective(destroyObeliskObjective)
+		Trigger.AfterDelay(DateTime.Seconds(5), function() Reinforce(MCVReinforcements) end)
+		ObeliskFlare.Destroy()
+		if AirSupportEnabled then AirSupport.Destroy() end
+	end)
+
+	Trigger.OnAllKilled(BiotechCenter, function()
+		player.MarkCompletedObjective(destroyBiotechCenterObjective)
+	end)
+	
+	Trigger.OnDamaged(Biolab, HuntTriggerFunction)
+	
+	AIRepairBuildings(enemy)
+	AIRebuildHarvesters(enemy)
+
+	Utils.Do(AttackTriggers, function(a)
+		Trigger.OnKilledOrCaptured(a, function()
+			NodVehicleProduction(Utils.Random(AutocreateSquads))
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(150), AutoCreateTeam)
+
+	NodInfantryProduction()
+
+	Camera.Position = UnitsRally.CenterPosition
+	ObeliskFlare = Actor.Create('flare', true, { Owner = player, Location = Flare.Location })
+	Reinforce(CommandoReinforcements)
+end
+
+Tick = function()
+	if DateTime.GameTime > DateTime.Seconds(5) and player.HasNoRequiredUnits() then
+		player.MarkFailedObjective(destroyBiotechCenterObjective)		
+	end
+end
+
+Reinforce = function(units)
+	Media.PlaySpeechNotification(player, "Reinforce")
+	ReinforceWithLandingCraft(units, lstStart.Location, lstEnd.Location, UnitsRally.Location)
+end
+
+ReinforceWithLandingCraft = function(units, transportStart, transportUnload, rallypoint)
+	local transport = Actor.Create("oldlst", true, { Owner = player, Facing = 0, Location = transportStart })
+	local subcell = 0
+	Utils.Do(units, function(a)
+		transport.LoadPassenger(Actor.Create(a, false, { Owner = transport.Owner, Facing = transport.Facing, Location = transportUnload, SubCell = subcell }))
+		subcell = subcell + 1
+	end)
+
+	transport.ScriptedMove(transportUnload)
+
+	transport.CallFunc(function()
+		Utils.Do(units, function()
+			local a = transport.UnloadPassenger()
+			a.IsInWorld = true
+			a.MoveIntoWorld(transport.Location - CVec.New(0, 1))
+
+			if rallypoint ~= nil then
+				a.Move(rallypoint)
+			end
+		end)
+	end)
+
+	transport.Wait(5)
+	transport.ScriptedMove(transportStart)
+	transport.Destroy()
+end
+
+HuntTriggerFunction = function()
+	local list = enemy.GetGroundAttackers()
+	Utils.Do(list, function(unit)
+		IdleHunt(unit)
+	end)
+end
+
+IdleHunt = function(unit)
+	if not unit.IsDead then
+		Trigger.OnIdle(unit, function()
+			unit.AttackMove(UnitsRally.Location, 50)
+			unit.Hunt()
+		end)
+	end
+end
+
+NodInfantryProduction = function()
+	if HandOfNod.IsDead or not HandOfNod.Owner == enemy then
```

Comment:
This doesn't work. It keeps building infantry after I captured it. Try 

```lua
if NodBarracks.Owner == player then
	return
```

instead